samples/MQTTSN/src/paho_iot_hub_telemetry_example_qos_0.c

```diff
@@ -0,0 +1,265 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+

```

Comment:
lets use code style mention in our sdk guide lines
[https://azure.github.io/azure-sdk/clang_implementation.html](https://azure.github.io/azure-sdk/clang_implementation.html)

Reply:
one good book that explains why we have coding style:
[https://www.oreilly.com/library/view/clean-code/9780136083238/](https://www.oreilly.com/library/view/clean-code/9780136083238/)

---
In reply to: [448060429](https://github.com/t-meitan-msft/azure-iot-udp-samples/pull/1#discussion_r448060429) [](ancestors = 448060429)

Reply:
You can add the `.clang-format` from the embedded SDK to make it easier